lessons/basics/functions.md

```diff
@@ -43,12 +43,11 @@ Elixir uses pattern matching to identify the first set of parameters which match
 
 ```elixir
 iex> handle_result = fn
-...>   {:ok, result} -> IO.puts "Handling result..."
```

Comment:
I think it would be better if we made use of the result instead of removing it.  What do you think?